Gopkg.toml

```diff
@@ -30,19 +30,19 @@ required = [
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "~6.0"
+  revision = "kubernetes-1.11.0"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.9.9"
+  version = "kubernetes-1.11.0"
 
 [[constraint]]
   name = "k8s.io/code-generator"
-  version = "kubernetes-1.9.9"
+  version = "kubernetes-1.11.0"
 
 [[override]]
   name = "github.com/json-iterator/go"
-  version = "1.1.4" # as used by apimachinery@kubernetes-1.9.9
+  version = "1.1.5" # as used by apimachinery@kubernetes-1.11.0
 
 [prune]
```

Comment:
I take it this is working for you? I only say that because I was running into all sorts of grouble getting code generation to work. See #79. I was able to get it to work adding the following to my `GoPkg.toml`:

```toml
[prune]
  go-tests = true
  unused-packages = true

  [[prune.project]]
    name = "k8s.io/code-generator"
    unused-packages = false
```

I was getting errors that the generator couldn't find the `boilerplate.go.txt` that comes with the code-generator repository.

Reply:
Me too - There is no `boilerplate.go.txt` within the vendor code-generator repository, so I added it manually (which is wrong) to unblock myself. I forgot to point this out, thanks for the reminder!

Reply:
Let me sync with @enisoc, and get back to you later. I believe your way of getting around this is better.

Reply:
@crimsonfaith91 I agree with the solution @rlguarino suggested. Can you try adding that?

@rlguarino Thanks for tracking that down!

Reply:
@enisoc Sure! I will include the `[prune]` section.